src/zjs_modules.c

```diff
@@ -24,7 +24,7 @@
 #include "zjs_timers.h"
 #include "zjs_util.h"
 #include "jerryscript-ext/module.h"
-#ifdef ZJS_ASHELL
+#if defined(ZJS_ASHELL) || defined(ZJS_DYNAMIC_LOAD)
 #include "ashell/file-utils.h"
```

Comment:
It's not very clean for ashell subdir to depend on things from src/ AND things in src/ to depend on things from ashell like this. I guess we missed it when it went in, but it would probably be better to move these file utils out to src.

And actually we have this exact same function already in zjs_fs.c called `file_exists`.

One problem as I just commented on IRC is how to use just a utility function like that without having to include the FS JS API which may not be needed.

Reply:
@grgustaf Where would you want it to live? I can just move file-utils to src/ or perhaps a new folder src/utils?

Reply:
I think we should say ashell depends on src/, not the other way around. Probably src/zjs_file_utils.c for now, and make sure that both zjs_fs.c and ashell use this same copy of the function.